bccsp/pkcs11/pkcs11.go

```diff
@@ -93,13 +95,16 @@ func (csp *impl) getSession() (session pkcs11.SessionHandle) {
 func createSession(ctx *pkcs11.Ctx, slot uint, pin string) pkcs11.SessionHandle {
 	var s pkcs11.SessionHandle
 	var err error
+	// attempt 10 times to open a session with a minor exponential backoff after each attempt
+	// backoff is 1.2^attempt * 100 in milliseconds
 	for i := 0; i < 10; i++ {
 		s, err = ctx.OpenSession(slot, pkcs11.CKF_SERIAL_SESSION|pkcs11.CKF_RW_SESSION)
 		if err != nil {
 			logger.Warningf("OpenSession failed, retrying [%s]\n", err)
 		} else {
 			break
 		}
+		time.Sleep(time.Duration(math.Pow(1.2, float64(i))*100) * time.Millisecond)
```

Comment:
Do we really need to bring in a math library to do
```
delay := 100 * time.Millisecond
for ... {
    ...
    time.Sleep(delay)
    delay = delay * 6 / 5
}
sleep = sleep * 6 / 5 ?
```
And adding more configuration leads to more doc and more pain. Just waiting 100ms between iterations would be a huge step forward here.

Reply:
Makes sense, exponential probably doesn't make much sense given how small of a scale it is anyways.